README.md

```diff
@@ -1,5 +1,7 @@
 # shell-functools
 
+[![Build Status](https://travis-ci.org/franklingu/shell-functools.svg?branch=master)](https://travis-ci.org/franklingu/shell-functools)
```

Comment:
Could you change these URLs to `../sharkdp/..`? I have just enabled Travis CI for this repo.